doc/install.rst

```diff
@@ -322,6 +321,68 @@ in a special config file $XDG_CONFIG_HOME/looking-glass/imgui.ini.
 Please refrain from editing this file manually as your changes might
 be overwritten by the client.
 
+.. _client_config_widget:
+
+Configuration Widget
+~~~~~~~~~~~~~~~~~~~~
+
+The Configuration Widget is accessible through the Overlay mode. The
+widget has multiple tabs that allow setting a variety of modes and
+parameters for the application.
+
+Settings tab
+^^^^^^^^^^^^
+
+- Performance Metrics: Enabling this will turn on the Metrics Graphs
+  Widget. Multiple graphs can be enabled and they will stack vertically.
+- EGL: Modify EGL features such as the algorithm used for scaling and
+  enabling night vision mode.
+Changes in the Settings tab are not persistent and will change back to
+their default values when the client is restarted.
+
+EGL Filters tab
+^^^^^^^^^^^^^^^
+
+The EGL Filters tab is a GUI for enabling, configuring and ordering
+the post-processing filter stack. Each filter can be expanded to open
+its settings. Filters can also be re-ordered by dragging them up or down.
+Filters application is cumulative, and order is important (e.g. applying CAS
+before FSR might have different results than the reverse). Users are
+encouraged to experiment with the order and parameters to achieve optimal
+results. The currently available filters include:
+
+- Downscaler: Filter for downscaling resolution. It can be used
+  to test how different input resolutions effect the filter stack and
+  find the best stack tuning for the application being used on the guest.
```

Comment:
The actual goal for the downscaler is to undo the poor upscaling used by games so we can upscale it better with FSR.

Reply:
I did not know that. Will push an updated section